src/main/java/no/entur/uttu/security/UserContextService.java

```diff
@@ -0,0 +1,23 @@
+package no.entur.uttu.security;
```

Comment:
In OTP similar abstractions are moved to a dedicated "spi" package (for example no.entur.uttu.security.spi) to stress the fact that this interface can be used to extend the product and that there is some guarantee of stability for the contract.